test/evergreen.yml

```diff
@@ -3616,6 +3616,29 @@ tasks:
       - func: "upload-perf-test-stats"
         vars:
           perf-test-name: 500m-btree-populate.wtperf
+      - func: "upload artifact"
+        vars:
+          upload_filename: WT_TEST.tgz
+          upload_source_dir: wiredtiger/cmake_build/bench/wtperf/WT_TEST_0_0/
+      - func: "cleanup"
```

Comment:
Just to confirm, this line is to avoid duplicated artifact upload in the post-task stage?

Reply:
If I do not call the cleanup function here then the post-task will try to archive it and upload it to the S3 bucket, which usually fails as there is a timeout setting and 15 minutes are wasted archiving (as the file is > 20G).

I put the cleanup function here so the post-task immediately fails which saves us 15 min of CPU time for each task.

Reply:
Thanks for the explanation. It would be nice to add a comment here.